web/src/frontend/src/digisos/skjema/personopplysninger/index.tsx

```diff
@@ -32,13 +44,31 @@ class Personopplysninger extends React.Component<Props, OwnProps> {
 					<FormattedMessage id="applikasjon.advarsel.gjenopptatt"/>
 				</Informasjonspanel>
 			</div>);
+		const systemdataEndretInfoPanel = (
+			<div className="skjema-sporsmal">
+				<Informasjonspanel
+					ikon={InformasjonspanelIkon.ELLA}
+					farge={DigisosFarge.VIKTIG}
+				>
+					<FormattedMessage id="oppsummering.systemdataendret.true"/>
+				</Informasjonspanel>
+			</div>);
+
+		if (skalSjekkeOmSystemdataErEndret){
+			return (
+				<div className="application-spinner">
+					<NavFrontendSpinner type="XXL" />
+				</div>
+			)
+		}
 
 		return (
 			<DigisosSkjemaSteg
 				steg={DigisosSteg.kontakt}
 				ikon={<William/>}
 			>
-				{this.props.gjenopptattSoknad && (gjennopptattSoknadInfoPanel)}
+				{this.props.gjenopptattSoknad && erSystemdataEndret === ErSystemdataEndret.NO && (gjennopptattSoknadInfoPanel)}
```

Comment:
fjerne "this.props", og ta ut gjenopptattSoknad på linje 37

gjenopptattSoknad kunne kanskje vært renamet til erGjenopptattSoknad? bare et forslag